README.md

```diff
@@ -26,17 +26,14 @@ developer communities.
 
 ## Features
 
-- 15+ [example Unity environments](docs/Learning-Environment-Examples.md)
-- Support for multiple environment configurations and training scenarios
```

Comment:
Why delete lines 30, 31 and 39?

Reply:
Because I did not like them anymore :) 
I can put them back

Reply:
ha - let's put them back.